src/qz/printer/action/PrintHTML.java

```diff
@@ -192,7 +192,7 @@ public void print(PrintOutput output, PrintOptions options) throws PrinterExcept
             settings.setJobName(pxlOpts.getJobName(Constants.HTML_PRINT));
             settings.setPrintQuality(PrintQuality.HIGH);
 
-            if (pxlOpts.getColorType() != null) {
```

Comment:
From what I could discern, pxlOpts.getColorType could never return null, so I removed it.

Reply:
It's rarely a good idea to remove a null guard.  I can see that the workflow sets an initial value on class construction, then uses a try/catch to set the value, so I agree with this sentiment, but in my tenure of programming it's been very rare that I've removed a NPE guard.